packages/liveblocks-core/src/room.ts

```diff
@@ -927,7 +927,7 @@ function makeStateMachine<
         id: token.actor,
         userInfo: token.info,
         userId: token.id,
-        isReadOnly: isReadOnly(token.scopes),
+        isReadOnly: isStorageReadOnly(token.scopes),
```

Comment:
Rather than adding a default value, would this behavior be clearer if we avoid even calling this in the case of an `undefined` value?

```suggestion
        isReadOnly: token.scopes !== undefined ? isStorageReadOnly(token.scopes) : false,
```

Reply:
The idea being that `user.scopes` never is going to be undefined, so we isolate that edge case, and only call the helper function in the expected case. We can later ensure that TS understands that it can never be undefined.